sandwich/src/main/java/com/skydoves/sandwich/ApiResponse.kt

```diff
@@ -51,7 +52,7 @@ sealed class ApiResponse<out T> {
     val statusCode: StatusCode = getStatusCodeFromResponse(response)
     val headers: Headers = response.headers()
     val raw: okhttp3.Response = response.raw()
-    val data: T? = response.body()
+    val data: T = response.body() ?: throw NoContentException(statusCode.code)
```

Comment:
Shouldn't this be `val data: T get() = response.body() ?: throw NotcontentException(statusCode.Code)` ? Because right not it will throw in the moment of `Success` creation, not in the moment of accessing the body

Reply:
Great point! I just created a new PR. (#23)